fluent-react/test/with_localization.test.js

```diff
@@ -100,31 +100,60 @@ bar = BAR {$arg}
     expect(getString("bar", {})).toBe("BAR {$arg}");
     expect(console.warn.mock.calls).toMatchInlineSnapshot(`
       Array [
+        Array [
+          "[@fluent/react] Error: The id \\"missing\\" did not match any messages in the localization bundles.",
+        ],
         Array [
           "[@fluent/react] ReferenceError: Unknown variable: $arg",
         ],
       ]
     `);
   });
 
-  test.skip("getString without access to the l10n context", () => {
+  test("getString with an empty bundle list", () => {
```

Comment:
It's still valid to provide an empty bundle list. To me this makes sense if for some reason you wanted to render you page with fallbacks. The LocalizationProvider still needs to be in place, but it will be configured with an empty list.